services/s3/src/it/java/software/amazon/awssdk/services/s3/s3express/DisableS3ExpressSessionAuthTest.java

```diff
@@ -0,0 +1,111 @@
+import static software.amazon.awssdk.testutils.service.S3BucketUtils.temporaryBucketName;
+
+
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
+import java.net.URI;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import software.amazon.awssdk.core.SdkSystemSetting;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.services.s3.S3Client;
+
+import software.amazon.awssdk.testutils.EnvironmentVariableHelper;
+
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+public class DisableS3ExpressSessionAuthTest extends S3ExpressIntegrationTestBase{
+
+    @RegisterExtension
+    static WireMockExtension s3WireMock = WireMockExtension.newInstance()
+                                                           .options(wireMockConfig().dynamicPort())
+                                                           .build();
+
+    private static final Region TEST_REGION = Region.US_EAST_1;
+    private static final String AZ = "use1-az4";
+    private static S3AsyncClient s3CrtAsync;
+    private static S3AsyncClient s3CrtAsyncDefault;
+    private static String testBucket;
+
+    private static final EnvironmentVariableHelper ENVIRONMENT_VARIABLE_HELPER = new EnvironmentVariableHelper();
+
+    private static final String S3EXPRESS_BUCKET_PATTERN = temporaryBucketName(S3ExpressIntegrationTest.class) +"--%s--x-s3";
+
+
+    private static String getS3ExpressBucketNameForAz(String az) {
+        return String.format(S3EXPRESS_BUCKET_PATTERN, az);
+    }
+
+    private static final String CREATE_SESSION_RESPONSE =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        + "<CreateSessionResult xmlns=\"http://s3.amazonaws.com/doc/2006-03-01/\">\n"
+        + "<Credentials><SessionToken>TheToken</SessionToken><SecretAccessKey>TheSecret"
+        + "</SecretAccessKey><AccessKeyId>TheAccessKey</AccessKeyId><Expiration>2025-02-19T00:23:59Z</Expiration></Credentials>\n"
+        + "</CreateSessionResult>";
+
+    @BeforeAll
+    static void setup() {
+        ENVIRONMENT_VARIABLE_HELPER.reset();
+        ENVIRONMENT_VARIABLE_HELPER.set(SdkSystemSetting.AWS_S3_DISABLE_EXPRESS_SESSION_AUTH.environmentVariable(), "false");
+
+        testBucket = getS3ExpressBucketNameForAz(AZ);
+
+        s3CrtAsyncDefault = s3CrtAsyncClientBuilder(TEST_REGION).endpointOverride(URI.create("http://s3.localhost.localstack.cloud:" + s3WireMock.getPort())).build();
+        s3CrtAsync = s3CrtAsyncClientBuilder(TEST_REGION).disableS3ExpressSessionAuth(true)
+                                                         .endpointOverride(URI.create("http://s3.localhost.localstack.cloud:" + s3WireMock.getPort())).build();
+
+    }
+
+    @AfterAll
+    static void clear(){
```

Comment:
We need to call `ENVIRONMENT_VARIABLE_HELPER.reset()` here so it doesn't affect any other tests